frameworks/PHP/swoole/swoole-server.php

```diff
@@ -38,15 +38,15 @@
     // Create an array with the response string.
     $arr = [];
 
-    $db->prepare('s', 'SELECT id, randomnumber FROM World WHERE id = $1');
+    $db->s = $db->s ?? $db->prepare('s', 'SELECT id, randomnumber FROM World WHERE id = $1');
```

Comment:
`$db->s` is useless and could be removed.

This is enouth, no?
```php
$db->prepare('s', 'SELECT id, randomnumber FROM World WHERE id = $1');
```

Reply:
@jcheron Thanks for the review, but`$db->s` is used to store states, it is initialized only once across multiple requests and added by purpose. 

There is no such API `$db->s->execute([$id]);`.

Same thing for the following suggestions.

Reply:
I can' t see where are used `$db->us`, `$db->uu` and `$db->s`...
The initialization of `$db->s` and others seems to be unused afterwards.

we can assume that this code below already does this initialization work for each worker:
```php
$db->prepare('s', 'SELECT id, randomnumber FROM World WHERE id = $1');
```

And that it is possible to run the prepared query `s` several times this way:
```php
$res = $db->execute('s', [$id]);
```

Reply:
Hi @jchannon, these variable is for checking if the `prepare` query is executed for once. For performance reason, these `prepare` query are only executed once for all the requests after the change. Hope this make sense?

Reply:
Hi @doubaokun 
I modified my previous comment after testing